admin/Makefile

```diff
@@ -25,13 +25,15 @@ LD_FLAGS = -ldflags " \
 			"
 
 release:                        ## Build pmm-admin release binary.
-	env CGO_ENABLED=0 go build -v $(LD_FLAGS) -o $(PMM_RELEASE_PATH)/pmm-admin
+	env CGO_ENABLED=0 go build -v $(LD_FLAGS) -o $(PMM_RELEASE_PATH)/pmm-admin ./cmd/pmm-admin/
```

Comment:
The release step does not need the `pmm` binary today. It will be required later when we're ready to release this to the public. I did not include it here just for the sake of not polluting the release process with unnecessary binaries.